ArduPlane/ArduPlane.cpp

```diff
@@ -772,12 +772,16 @@ void Plane::update_flight_mode(void)
         nav_roll_cd  = (channel_roll->get_control_in() / 4500.0) * roll_limit;
         nav_roll_cd = constrain_int32(nav_roll_cd, -roll_limit, roll_limit);
         float pitch_input = channel_pitch->norm_input();
-        if (pitch_input > 0) {
-            nav_pitch_cd = pitch_input * MIN(aparm.pitch_limit_max_cd, quadplane.aparm.angle_max);
+        if (quadplane.is_tailsitter()) {
+            nav_pitch_cd = pitch_input * quadplane.aparm.angle_max;
```

Comment:
comments added; note that this is scaling from normalized units to centidegrees, not constraining the PWM output limits